configs/service.yaml

```diff
@@ -15,6 +15,13 @@ redis_max_retry_backoff: "512ms"
 
 blob_max_inline_size: 65536
 
+grpc_keepalive_enabled: false
+grpc_keepalive_max_connection_idle: 300
```

Comment:
Use time.Duration for the fields and use the following default values:

grpc_keepalive_max_connection_age: "12h"
grpc_keepalive_max_connection_age_grace: "5m"
grpc_keepalive_time: "1h"
grpc_keepalive_timeout: "20s"